exercises/concept/pacman-rules/src/PacmanRules.hs

```diff
@@ -0,0 +1,17 @@
+module PacmanRules (eatsGhost, scores, loses, wins) where
+
+eatsGhost :: Bool -> Bool -> Bool
+eatsGhost =
```

Comment:
The Elixir original includes parameter names. I am in favor of including them here as well. After all, it is an introductory exercise about _booleans_, not _function declarations_.